addons/core/addon/components/dropdown/index.hbs

```diff
@@ -35,7 +40,11 @@
         checked={{includes itemOption.id @checkedItems}}
         {{on 'change' this.selectItem}}
       >
-        {{itemOption.displayName}}
+        {{#if itemOption.name}}
+          {{itemOption.name}}
+        {{else}}
+          {{itemOption.id}}
+        {{/if}}
```

Comment:
```suggestion
        {{#if itemOption.name itemOption.name itemOption.id}}
```

You also need to update this in the targets controller where we make objects with `displayName`